solana/solana-ibc/programs/solana-ibc/src/error.rs

```diff
@@ -129,6 +129,9 @@ pub enum Error {
     /// blocks would never get finalized until more stake is added and quorum
     /// stake is less than head stake.
     MinQuorumStakeHigherThanTotalStake,
+
+    /// When a connection id which is not yet created is updated
+    InvalidConnectionId,
```

Comment:
Would it make sense to use https://docs.rs/ibc/latest/ibc/core/connection/types/error/enum.ConnectionError.html#variant.ConnectionNotFound instead?

Reply:
i think it would.